### PR TITLE
remove duplicate package references

### PR DESCRIPTION
**Pull request by mjkkirschner**
found some duplicate refs I added, cleaning them up

### CODE REVIEW

File: src/Libraries/CoreNodeModels/CoreNodeModels.csproj [hunk 1]
```diff
@@ -26,17 +26,11 @@
       <!--Exclude copying the runtime dlls because we will handle it in the BinaryCompatibilityOps target -->
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
-	  <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
     <PackageReference Include="CoreCLR-NCalc" Version="2.2.92" />
   </ItemGroup>
   <ItemGroup>
-  <PackageReference Include="ncalc" Version="1.3.8">
```
Comment (aparajit-pratap) [inline src/Libraries/CoreNodeModels/CoreNodeModels.csproj:35]: It looks like if you remove `ncalc` it is now only present for net48.

Comment (mjkkirschner) [reply src/Libraries/CoreNodeModels/CoreNodeModels.csproj:35]: see above - `CoreCLR-NCalc` was added.

File: src/Libraries/CoreNodeModels/CoreNodeModels.csproj [hunk 2]
```diff
@@ -26,17 +26,11 @@
       <!--Exclude copying the runtime dlls because we will handle it in the BinaryCompatibilityOps target -->
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
-	  <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```
Comment (pinzart90) [inline src/Libraries/CoreNodeModels/CoreNodeModels.csproj:29]: why remove Newtonsoft from net48?
Maybe you can create a common ItemGroup for both net48 and net6.0 and put it there ?

File: src/Libraries/CoreNodeModels/CoreNodeModels.csproj [hunk 3]
```diff
@@ -26,17 +26,11 @@
...
   </ItemGroup>
   <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
     <PackageReference Include="CoreCLR-NCalc" Version="2.2.92" />
   </ItemGroup>
   <ItemGroup>
-  <PackageReference Include="ncalc" Version="1.3.8">
-    <GeneratePathProperty>true</GeneratePathProperty>
-    <!--Exclude copying the runtime dlls because we will handle it in the BinaryCompatibilityOps target -->
-    <ExcludeAssets>runtime</ExcludeAssets>
-  </PackageReference>
```
Comment (pinzart90) [inline src/Libraries/CoreNodeModels/CoreNodeModels.csproj:39]: I think the ItemGroup with all the `References` to PresentationCore etc should be under net48

Comment (pinzart90) [reply src/Libraries/CoreNodeModels/CoreNodeModels.csproj:39]: Also I am not sure why we have WPF and Windows references in a core project. 
I hope these are just junk references that we can remove. If not we need to segregate the wpf and windows only code in the project

File: src/Libraries/CoreNodeModels/CoreNodeModels.csproj (hunk 2 above)
Comment (mjkkirschner) [reply src/Libraries/CoreNodeModels/CoreNodeModels.csproj:29]: it's referenced below on line 40.

File: src/Libraries/CoreNodeModels/CoreNodeModels.csproj (hunk 3 above)
Comment (mjkkirschner) [reply src/Libraries/CoreNodeModels/CoreNodeModels.csproj:39]: not sure about the WPF parts but I don't think that has been changed in years.

**Comment by mjkkirschner** [comment]
@pinzart90 I was able to remove the WPF / windows refs and the solutions built fine for me locally. PTAL

**Review by aparajit-pratap — APPROVED** [review]

**Pull request merged**